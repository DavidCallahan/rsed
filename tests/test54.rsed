text="1,2,3,4"
list=split $text with ","
print $list[0]
print $list[3]
text="a.b.c.d"
list= split $text columns 2 4 6
print $list[0]
print $list[3]
