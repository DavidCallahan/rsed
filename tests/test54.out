1
4
a.
d
